core/src/main/java/com/linecorp/armeria/server/FallbackService.java

```diff
@@ -77,7 +84,16 @@ private static HttpResponse handleNotFound(ServiceRequestContext ctx, RoutingCon
                                               .build());
     }
 
-    private static HttpResponse newHeadersOnlyResponse(HttpStatus status) {
+    private static HttpResponse newFallbackResponse(ServiceRequestContext ctx, HttpStatus status) {
+        final ServiceErrorHandler errorHandler = ctx.config().errorHandler();
+        final AggregatedHttpResponse rendered = errorHandler.renderStatus(ctx.config(),
+                                                                          ctx.request().headers(),
+                                                                          status,
+                                                                          null,
+                                                                          null);
+        if (rendered != null) {
```

Comment:
Note: `rendered` can still be `null` if `ServiceErrorHandler.orElse` is overridden. I think the current null check is fine 👍

Reply:
Hope we can have `final` for the method. 😉